package.json

```diff
@@ -4,6 +4,7 @@
   "description": "This is a proposed template repository for applications for future developments. This will help get you set up as you build your application.",
   "main": "index.js",
   "dependencies": {
+    "@reach/skip-nav": "^0.15.2",
```

Comment:
I think I installed this dependency as well, oh well.

Reply:
It wasn't installed on the branch I was working on, so needed to install here otherwise I got errors. Should be fine installing twice :)